packages/vue/src/components/sign-in.vue

```diff
@@ -1,23 +1,27 @@
 <script setup lang="ts">
-import { computed, ComputedRef, useAttrs } from 'vue';
+import { computed, toRefs, useAttrs } from 'vue';
 
 import {
+  AuthenticatorServiceFacade,
   authenticatorTextUtil,
-  getActorState,
   getFormDataFromEvent,
-  SignInState,
   translate,
 } from '@aws-amplify/ui';
 
 import FederatedSignIn from './federated-sign-in.vue';
 
 // @xstate
-import { useAuth, useAuthenticator } from '../composables/useAuth';
+import { useAuthenticator } from '../composables/useAuth';
 import BaseFormFields from './primitives/base-form-fields.vue';
 
-const props = useAuthenticator();
+// `useAuthenticator` is casted for temporary type safety on this file.
```

Comment:
Why is it temporary, do you plan to remove it because it creates breaking type changes?

Reply:
Yeah, ideally the long term fix is fully type `useAuthenticator` but that's a technically a breaking change because it is currently typed as `any`. 

This `useAuthenticator` usage is only internal, so I temporarily casted the type so that we can at least be confident about the refactors we are making.